uw-frame-components/portal/widgets/controllers.js

```diff
@@ -181,6 +181,13 @@ define(['angular'], function(angular) {
       if (!$scope.config.showShowing) {
         $scope.config.showShowing = false;
       }
+      // Set sensible fallbacks in case widget config would create display problems.
+      if ($scope.config.lim > 6) {
+        $scope.config.lim = 6;
+      }
+      if ($scope.config.titleLim > 50) {
```

Comment:
:thought_balloon: Should this be checked in a JSON schema type validation, instead of at run time?